test/language_server/completion_engine_test.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+require "test_helper"
+
+module ThemeCheck
+  module LanguageServer
+    class CompletionEngineTest < Minitest::Test
```

Comment:
Not sure if it's relevant. Should there be some tests w/ missing closing tokens at this level? Missing: `%}`, `}}` ? Worth it? I see it is tested bellow.

Reply:
Definitely, since this is the kind of thing that bit me during the demo video :p To be fixed soon :)